core/domain/expression_parser.py

```diff
@@ -124,7 +124,7 @@ def tokenize(expression):
     # ['x','+','e','*','psi','*','l','*','o','*','n']. a^2.
     re_string = r'(%s|[a-zA-Z]|[0-9]+\.[0-9]+|[0-9]+|[%s])' % (
         '|'.join(sorted(
-            constants.GREEK_LETTER_NAMES_TO_SYMBOLS.keys() +
+            list(constants.GREEK_LETTER_NAMES_TO_SYMBOLS.keys()) +
```

Comment:
Why is this needed? The sorted function can take in `dict_keys` and it will return a `list`.

Reply:
We need this because you cannot do `+` to `dict_keys` and `list`. `TypeError: unsupported operand type(s) for +: 'dict_keys' and 'list'`